test/local/log.js

```diff
@@ -108,7 +108,6 @@ describe('log', () => {
       assert.equal(args.length, 2, 'logger.error was passed two arguments')
       assert.equal(args[0], 'log.activityEvent', 'first argument was function name')
       assert.deepEqual(args[1], {
-        op: 'log.activityEvent',
```

Comment:
I remember these annoying me in the past, that we had to assert for the presence of the property which wasn't being used. Nice to see them disappear.